edgar-app/src/components/dashboardPages/documents/modal/AddDocumentModalContent.tsx

```diff
@@ -0,0 +1,51 @@
+import { FormLabel, HStack, Input, Select, VStack } from '@chakra-ui/react';
+import ErrorMessage from 'components/forms/ErrorMessage';
+import { FieldErrors, UseFormRegister } from 'react-hook-form';
+import { AddDocumentType } from 'types/dashboard/documents/AddDocumentType';
+
+const AddDocumentModalContent = ({
+	register,
+	errors,
+}: {
+	register: UseFormRegister<AddDocumentType>;
+	errors: FieldErrors<AddDocumentType>;
+}): JSX.Element => (
+	<VStack spacing="24px" w="100%" px="32px">
+		<VStack w="100%" align="start">
+			<FormLabel size="boldLg">Votre document</FormLabel>
+			<Input
+				{...register('document', { minLength: 1, maxLength: 50, required: true })}
```

Comment:
No need to add a `minLength` and a `maxLength` because it's a file input and not a text input